### DISCUSSION TITLE
add "font-feature-settings" css option

### DISCUSSION THREAD

**Discussion by zina0316** [monkeytypegame/monkeytype#5122] [Ideas]
My vscode and monkeytype both using the "Fira Code" font

this is how my monkeytype text looks
![brave_d6WP5kjvma](https://github.com/monkeytypegame/monkeytype/assets/94322065/bf60077f-9967-413b-88bf-6c304ae99f3b)

and this is how my vscode looks
![Code_UpfPCdeaUr](https://github.com/monkeytypegame/monkeytype/assets/94322065/7816f1e9-05fc-433e-9dbd-d104217fa9ac)

I like the way my vscode shows "a" and "g" (see language)

the way I done it is to insert `font-feature-settings: '"cv01", "cv02"'` in the setting.json file

I wish I can also do this in monkeytype and don't need to try other fonts

**Comment by Miodec** [comment]
Sorry, I don't think it will be used by enough people to add this. I would recommend using a custom css extension instead.
